no-littering.el

```diff
@@ -262,6 +262,9 @@ directories."
     (setq helm-adaptive-history-file       (var "helm/adaptive-history.el"))
     (setq helm-github-stars-cache-file     (var "helm/github-stars-cache.el"))
     (setq historian-save-file              (var "historian-save.el"))
+    (eval-after-load 'indium
+      `(make-directory ,(var "indium/") t))
+    (setq indium-workspace-file (var "indium/indium-workspaces.el"))
```

Comment:
Do you need a directory if it's for one file only? If you want the directory, should the file be named "workspace.el" instead of "indium-worspace.el"?